homeassistant/components/tibber/__init__.py

```diff
@@ -53,6 +53,11 @@ async def _close(event: Event) -> None:
 
     try:
         await tibber_connection.update_info()
+        if tibber_connection.name == "":
+            msg = "Could not fetch Tibber data. Requesting retry."
+            _LOGGER.warning(msg)
```

Comment:
Don't log the warning. The exception argument will be logged at warning level by default and will be deduplicated.

Reply:
Yes, I saw that but it will be a log line from the HA core and I thought it's a good idea to have it logged as tibber component too. What do you think?

Reply:
No, we want this to be handled by our helper in core.